packages/providers/src/provider.ts

```diff
@@ -247,8 +247,13 @@ export default class Provider {
           });
   }
 
-  static chainInfoCache: ChainInfoCache = {};
-  static nodeInfoCache: NodeInfoCache = {};
+  static resetChainAndNodeCaches() {
```

Comment:
`clearChainAndNodeCaches` might be a better name because `reset` might imply a previous initial state to which one can reset, whereas the implementation fully clears out the cache.